### ISSUE TITLE
.conf `stream_compress_adapt` documentation and behaviour

### ISSUE THREAD

**Issue by HaleTom** [digint/btrbk#551]
The .conf man page says:

```
       stream_compress_adapt default|<number>
           Enable adaptive compression for <compress_command>. Only supported
           for "zstd" (version >= 1.3.6).
```

But the [code](https://github.com/digint/btrbk/blob/a4d4dff0d12489137aae3d4b629f9c5a5764593c/btrbk#L118C3-L118C89) says:

    stream_compress_adapt       => { default => undef,         accept => [qw( yes no )] },

From this, I can't work out what the default is, but it's certainly not a number as per the manpage.

This code seems to pass on a "yes" or "no": https://github.com/digint/btrbk/blob/a4d4dff0d12489137aae3d4b629f9c5a5764593c/btrbk#L757-L761

Whereas, [zstd manual](https://manned.org/zstd#:~:text=%2D%2Dadapt%5B%3Dmin%3D%23%2Cmax%3D%23%5D) indicates that it wants:

    --adapt[=min=#,max=#]

**Comment by digint** [comment]
Thanks, the documentation was wrong from the beginning on (see #397), fixed in  18ddc659790de489967c2c82213fb5fba7955d40.

For now, btrbk does not support min/max: probably nobody uses this, do you?

**Comment by HaleTom** [comment]
On or off is fine for me.

Thanks a bunch!
